src/SimpleThings/EntityAudit/EventListener/LogRevisionsListener.php

```diff
@@ -199,6 +199,13 @@ public function postFlush(PostFlushEventArgs $eventArgs)
                 $this->em->getConnection()->executeQuery($sql, $params, $types);
             }
         }
+        $this->cleanUp();
+    }
+
+    private function cleanUp()
+    {
+        $this->extraUpdates = array();
```

Comment:
For a tactical implementation, this is perfectly acceptable.

However, for future work (log a task/sub-task in Jira) I would suggest applying something on line `123`:
```
if (! isset($updateData[$meta->table['name']]) || ! $updateData[$meta->table['name']]) {
    continue;
}
````

By this point, we've already added this as an extra update as a result of lines `279` and `287` because they're determined to be related entities that are audited.  However, at this point in the life cycle (post-flush), if we haven't encountered a need to update them, they should be removed from `$this->extraUpdates` at that point.